commit ccb1d828059bea0f8d471a1277d13c0625251b9b
Author: nx111 <gd.zhangdz@gmail.com>
Date:   Wed Apr 2 02:05:39 2014 +0800

    Revert "fix the mac build"
    
    This reverts commit 43299c39a8f4f055b23dc7bb9827eebe29da15df.

diff --git a/media/java/android/media/RingtoneManager.java b/media/java/android/media/RingtoneManager.java
index 5439688..48e5b62 100644
--- a/media/java/android/media/RingtoneManager.java
+++ b/media/java/android/media/RingtoneManager.java
@@ -34,7 +34,6 @@ import android.provider.Settings;
 import android.provider.Settings.System;
 import android.util.Log;
 
-import java.io.File;
 import java.util.ArrayList;
 import java.util.List;
 
@@ -225,8 +224,6 @@ public class RingtoneManager {
      */
     public static final int URI_COLUMN_INDEX = 2;
 
-    private static final Uri BAHHH = Uri.fromFile(new File("/system/media/audio/ringtones/Sheep.mp3"));
-
     private Activity mActivity;
     private Context mContext;
 
@@ -653,7 +650,7 @@ public class RingtoneManager {
                 }
             }
 
-            r.setUri(BAHHH);
+            r.setUri(ringtoneUri);
             return r;
         } catch (Exception ex) {
             Log.e(TAG, "Failed to open ringtone " + ringtoneUri + ": " + ex);
